src/Castle.Facilities.AutoTx.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Security;
+[assembly: AssemblyTitle("Castle AutoTx Facility")]
+[assembly: AssemblyDescription("This facility enables ambient easy-to-use declarative transactions through the [Transaction] attribute. This is the glue that holds the bits and pieces from Castle Transactions together and the stuff that will make your co-workers googly-eyed and impressed.")]
+[assembly: AssemblyProduct("Castle AutoTx Facility")]
+[assembly: AssemblyCopyright("Apache 2.0 Licensed. (c) The Castle Project 2004-2011")]
+[assembly: AssemblyVersion("3.3.0.14213")]
```

Comment:
As mentioned elsewhere it would be great to get this project working like Castle Core and Windsor which automatically pick up their version from Git tags.